src/connectors/CassandraConnector.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const Promise = require('bluebird');
+const cassandra = Promise.promisifyAll(require('cassandra-driver'));
+
+const CASSANDRA_CONTACT_POINTS = process.env.CASSANDRA_CONTACT_POINTS;
+
+const options = {
+  contactPoints: [CASSANDRA_CONTACT_POINTS],
+  keyspace: process.env.CASSANDRA_KEYSPACE
+};
+
+module.exports = {
```

Comment:
It looks as though this PR will take a while longer to get merged. As such, could you please split out the Cassandra-related functionality into a separate PR so that folks working on the resolvers can leverage this work?